app/client/src/components/pages/LocationMap/index.js

```diff
@@ -217,6 +227,7 @@ function LocationMap({ layout = 'narrow', windowHeight, children }: Props) {
       providersLayer,
       boundariesLayer,
       monitoringStationsLayer,
+      upstreamLayer,
       issuesLayer,
       dischargersLayer,
       nonprofitsLayer,
```

Comment:
```suggestion
      providersLayer,
      boundariesLayer,
      upstreamLayer,
      monitoringStationsLayer,
      issuesLayer,
      dischargersLayer,
      nonprofitsLayer,
```

I think the upstreamLayer should be displayed below the monitoring station layer, so that boundaries are always below the features.

Reply:
Changed, thanks!